source/_docs/installation.markdown

```diff
@@ -42,18 +42,18 @@ NUC i7/i9 | Pure power, you should not have *any* performance issues
 
 ## Recommended
 
-These install options are fully supported by Home Assistant's documentation. For example, if an integration requires that you install something to make it work on one of these methods then the integration page will document the steps required.
+Using the Home Assistant OS method is the fully supported installation option by Home Assistant's documentation. For example, if an integration requires that you install something to make it work then the integration page will document the steps required. The Home Assistant Container method, on the other hand, may have limited documentation available, for example if you want to install and use add-ons.
 
 <div class='note'>
 
-The only installation methods that allow you to use Home Assistant Add-ons is using the Home Assistant image. All other methods only install the base Home Assistant packages, however the software from the add-ons may still usually be installed manually like any other program.
+The only installation method that allows you to use Home Assistant Add-ons is using the Home Assistant OS method. All other methods (including Home Assistant Container) only install the base Home Assistant packages, however the software from the add-ons may still usually be installed manually like any other program.
```

Comment:
Incorrect, Supervised is another method that provides add-ons. Add-ons are not limited to the OS method.

Reply:
Ah, right. I think I'm getting confused between OS and Supervised. I meant to only distinguish between the OS and Container methods, since they are the ones in the "Recommended" section.

Would it be correct to say something along the lines of this?
> "While the Home Assistant OS method allows you to install and use Home Assistant Add-ons directly from the Web GUI, the Home Assistant Container method does not."

Reply:
You can read more about this in ADR-0012, we have 4 supported installation methods.

Reply:
Sorry, I'm at a loss here. What's ADR-0012? I find nothing relevant when searching here on GitHub, on https://www.home-assistant.io/ or on Google.

Reply:
If I changed this part to the following, would that be a reasonable improvement to the text?

> The only installation methods that allows you to use Home Assistant Add-ons are using the Home Assistant OS or Home Assistant Supervised methods. All other methods (including Home Assistant Container) only install the base Home Assistant packages, however the software from the add-ons may still usually be installed manually like any other program.